client/CServerHandler.cpp

```diff
@@ -454,11 +454,11 @@ void CServerHandler::setPlayer(PlayerColor color) const
 	sendLobbyPack(lsp);
 }
 
-void CServerHandler::setPlayerOption(ui8 what, si8 dir, PlayerColor player) const
+void CServerHandler::setPlayerOption(ui8 what, si16 value, PlayerColor player) const
 {
 	LobbyChangePlayerOption lcpo;
 	lcpo.what = what;
-	lcpo.direction = dir;
+	lcpo.value = value;
```

Comment:
why not keeping both? Those small arrows are still visible, so could be both ways to control (arrows + clicking at icon)

Reply:
This will keep both. But use the same variable for both. `direction` is misleading then.